packages/storybook/stories/inbox/with-toast.stories.tsx

```diff
@@ -99,18 +104,22 @@ export function AsyncUserId() {
   );
 }
 
-export function WithTheme() {
+export function OnMessage() {
+  const handleOnMessage = (message) => {
+    console.log(message);
```

Comment:
probably remove log?

Reply:
eh, its in a story

Reply:
ill be revisting all of these stories soon